src/deepsparse/transformers/pipelines/text_generation/parse_inputs.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from deepsparse.operators import Operator
+from deepsparse.transformers.schemas.text_generation_schemas import (
+    GenerationDefaults,
+    TextGenerationInput,
+)
+from deepsparse.utils import InferenceState, PipelineState
+
+
+class ParseInputs(Operator):
```

Comment:
can we name this a bit more specific for text generation? and is there anything we can add more generically so we don't need to require kwargs by default?

Reply:
I updated the name. 
For requiring kwargs by default, do you mean the default Operator functionality? Or specific to text_generation?